packages/plugin-conversation/src/encrypter.js

```diff
@@ -9,7 +9,7 @@ import {isArray, isFunction, isString} from 'lodash';
 import S from 'string';
 
 const encryptableActivities = [
-  `add`,
+  `assign`,
```

Comment:
We've discussed this - The assign verb is an unimplemented conversation service feature and not a teams feature. I'll take it out of this array until it gets implemented.